test/integration/color-modifier-references.test.ts

```diff
@@ -43,14 +43,17 @@ describe('typography references', () => {
 
   it('supports references inside color modifiers', async () => {
     const file = await promises.readFile(outputFilePath, 'utf-8');
-    expect(file).to.include(
-      `--sdAlpha: 0.3;
-  --sdColor: #ffffff4d;`,
-    );
+    const content = excerpt(file, { before: ':root {', after: '--sdModifier' });
+    const normalizeWhitespace = (str: string) => str.replace(/^\s+/gm, ''); // Remove leading spaces/tabs
```

Comment:
doesn't `String.trim()` do the same thing and also tackle trailing spaces? I think the excerpt util handles trimming leading/trailing whitespace already right? And if not, I feel like it should, rather than having to do it here in the test